RELEASE_NOTES.md

```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## Version TBD
+_Date TBD_
+
+### Table Modifications
+* Add participant_gender_statistics table
+* Add participant_age_statistics table
+* Add participant_education_statistics table
+* Add participant_experience_statistics table
```

Comment:
Usually try to write these as things that are helpful for an end user, or at least have a top-line description that is. Maybe something like:
Added tables to support statistics related to SED Participant demographics:
* Gender
* Age
* Education Type
* Experience Levels

Reply:
I'll make this update.